### ISSUE TITLE
better integration of mapview

### ISSUE THREAD

**Issue by dylanbeaudette** [ncss-tech/soilReports#46]
http://ncss-tech.github.io/soil-pit/projects/MLRA111B_Glynwood_B-slope_Erosion_Northeastern_IN.html

**Comment by brownag** [comment]
I see the use case of mapview here, but what would make it "better integrated"?

**Comment by dylanbeaudette** [comment]
Good point, poor choice of words on my part. Maybe more widely used / linkages to what is displayed in the mapview and what is displayed elsewhere in the report.
